conda/recipes/cudf/meta.yaml

```
@@ -83,8 +81,8 @@ requirements:
     - pandas >=2.0,<2.2.3dev0
     - cupy >=12.0.0
     - numba >=0.57
-    - numpy >=1.23,<3.0a0
-    - {{ pin_compatible('pyarrow', max_pin='x.x') }}
+    - numpy >=1.23,<2.0a0
```

This should still be NumPy 3 as the upper bound

```suggestion
    - numpy >=1.23,<3.0a0
```